test/model/workloads/WT-13618.workload

```diff
@@ -0,0 +1,13 @@
+create_table(4, "table4", "r", "Q")
+begin_transaction(816)
+begin_transaction(743)
```

Comment:
better to interchange the "being_transaction" calls at line 2 & 3, simulating an increasing order for better readability as it looks more natural.

Reply:
@bvpvamsikrishna, I see your point but it's a backport. I would rather not change this.

Reply:
ok